core/templates/base-components/base-content.component.css

```diff
@@ -1,8 +1,9 @@
 /*
-  NOTE to developers: Only the '.css' files should be edited. The styling should
+  NOTE to developers: Only the files ending with '.css' (but not '.rtl.css') should be edited. The styling should
   be generalized enough that the autogenerated '.rtl.css' files should work
   without any manual input.
-  This can be generated by running 'python -m scripts.generate_rtl_css' script.
+  The '.rtl.css' file can be generated by running
+  'python -m scripts.rtl_css --mode generate' script.
```

Comment:
Sorry, I missed this previously, but there should be a "the" after "running" (here and elsewhere).

Reply:
Done